governance/api.go

```diff
@@ -96,6 +98,22 @@ func (api *PublicGovernanceAPI) Vote(key string, val interface{}) (string, error
 			return "", errRemoveSelf
 		}
 	}
+	if strings.ToLower(key) == "kip71.lowerboundbasefee" {
+		if _, ok := api.governance.ValidateVote(&GovernanceVote{Key: key, Value: val}); !ok {
+			return "", errInvalidKeyValue
+		}
+		if val.(uint64) > api.governance.UpperBoundBaseFee() {
+			return "", errInvalidLowerBound
+		}
+	}
+	if strings.ToLower(key) == "kip71.upperboundbasefee" {
+		if _, ok := api.governance.ValidateVote(&GovernanceVote{Key: key, Value: val}); !ok {
+			return "", errInvalidKeyValue
+		}
+		if val.(uint64) < api.governance.LowerBoundBaseFee() {
+			return "", errInvalidUpperBound
+		}
+	}
```

Comment:
I think you are considering something like https://github.com/klaytn/klaytn/pull/1379#issuecomment-1138077733
If the `kip71.gastarget` is zero or very small number, then It can always cause the `basefee` to be set in `kip71.upperboundbasefee`. It's uneasy to constrain the parameters. Do you have any idea for this?

Reply:
`feedominator`:
- Must not be zero
- Must be `e` to 1, where `e` > 0
`blockgaslimit`: 
- Do not need to provide setter API. `Denomiator` already stands for suppressing an unexpectedly surprising change of the baseFee. I would appreciate it if let me know the need for this value. 
gastarget
- Do not need to provide setter API since it is currently a fixed value and not related to increase or decrease by the new policy. I recommend reconsidering the alternative way. For example, the `gastarget` can be an average gasUsed of near `n` blocks, rather fixes static value.

Reply:
I'd like to remain it another issue. We need to discuss it after forward. Thanks.

Reply:
Okay. Let me note another comment before resolving it. 
- The operation of `feedominator` seems better to substitute from division to multiply. If the operation is 'division', the decision of its value seems not easy. If the operator is 'multiply', the value can be obviously zero to one.
- The name `feedominaotr` seems awkward. Refer to other notations for the scaling factor in well-known standards.